### ISSUE TITLE
1.1.1 not backwards compatible

### ISSUE THREAD

**Issue by jamesw-minted** [iarna/wide-align#51]
the bump between 1.1.0 and 1.1.1 for `wide-align`, according to semver, should be a backwards compatible change. however, the version of `string-width` has been bumped from 1.x to 2.x at https://github.com/iarna/wide-align/commit/0909e69a807f174396cdeaa10814966203b3f8d0. according to semver, this is a non-backwards compatible change. the appropriate bump would've been for wide-align to bump the major version to signify that the changes are not backwards-compatible.

i came across this package since it's a dependency of popular sass loader by node-sass v3.x:
node-sass -> npmlog -> gauge -> wide-align. the 1.1.1 bump broke a build system we were using that relied on node-sass with node 0.12, which did not natively support the `const` syntax.

```
17:25:56 /local/workspace/project.package_react_apps/node_modules/gulp-sass/node_modules/node-sass/node_modules/npmlog/node_modules/gauge/node_modules/wide-align/node_modules/string-width/index.js:2
17:25:56 const stripAnsi = require('strip-ansi');
17:25:56 ^^^^^
17:25:56 SyntaxError: Use of const in strict mode.
17:25:56     at exports.runInThisContext (vm.js:73:16)
17:25:56     at Module._compile (module.js:443:25)
17:25:56     at Object.Module._extensions..js (module.js:478:10)
17:25:56     at Module.load (module.js:355:32)
17:25:56     at Function.Module._load (module.js:310:12)
17:25:56     at Module.require (module.js:365:17)
17:25:56     at require (module.js:384:17)
17:25:56     at Object.<anonymous> (/local/workspace/project/node_modules/gulp-sass/node_modules/node-sass/node_modules/npmlog/node_modules/gauge/node_modules/wide-align/align.js:2:19)
17:25:56     at Module._compile (module.js:460:26)
17:25:56     at Object.Module._extensions..js (module.js:478:10)
```

**Comment by brianatwhistle** [comment]
This is happening to me too, trying to build node-sass@3.13.1.

**Comment by jamesw-minted** [comment]
for anyone in this thread who can afford to, not-using an ancient version of npm (0.12) does the trick (i tested it with npm=lts/boron). alternatively, if you have a previously installed node_modules folder, you can also try generating a npm shrinkwarp file and manually lock down the subdependency version. for me, our build process is stuck on 0.12.x for a while, so i ended up forking the dependency chain and locking the versions in my local fork.

**Comment by daleconboy** [comment]
Same here. Looks like it's due to the major bump of the `string-width` dependency.

https://github.com/iarna/wide-align/commit/0909e69a807f174396cdeaa10814966203b3f8d0

**Comment by kgryte** [comment]
Encountering the same issue.

**Comment by jamesw-minted** [comment]
for anyone else stuck on this thread: i was able to work around the issue by locking down the version in my project's `package.json` by adding `wide-align` as a dependency:

```
"dependencies": {
  ...
  "gulp-sass": "^3.0.0",
  "wide-align": "1.1.0"
}
```

this works because `gauge` only specifies ^1.1.0, and npm can satisfy both requirements with one version (1.1.0) package.

**Comment by casper-rasmussen** [comment]
We're having the same issue.

**Comment by yesman82** [comment]
Locking down dependencies (e.g. with shrinkwrap) not possible with node versions < version 6!
Mine: v0.10.25

**Comment by alarre** [comment]
Same here! Stopped working overnight.

Will try suggested solution, just wanted to say that if you have "node-inspector", "v8-debug", "node-pre-gyp", etc, install will fail.

**Comment by wparam** [comment]
Same here, using @jamesw-minted solution solved my issue, thanks

**Comment by iarna** [comment]
Yeah, not intended, I'll revert that!

**Comment by kgryte** [comment]
Awesome! Thank you, @iarna!

**Comment by iarna** [comment]
1.1.2 published with `string-width@1.0.2`
